requirements/dev.txt

```diff
@@ -150,7 +150,7 @@ requests-toolbelt==0.10.1
     # via twine
 rfc3986==2.0.0
     # via twine
-rich==12.6.0
+rich==13.0.0
```

Comment:
Requires `Python>=3.7` now. For details, see https://github.com/Textualize/rich/blob/master/CHANGELOG.md#1300---2022-12-30